test/test-server/quotas_handler.go

```diff
@@ -19,45 +19,45 @@ func handleAppliedQuotas(t *testing.T) http.HandlerFunc {
 		quotaCode := r.URL.Query().Get("id")
 
 		quota1 := servicequotav1.ServiceQuotaV1AppliedQuota{
-			Id:           stringToPtr("quota_a"),
-			Scope:        stringToPtr("kafka_cluster"),
-			DisplayName:  stringToPtr("Quota A"),
+			Id:           ptr("quota_a"),
+			Scope:        ptr("kafka_cluster"),
+			DisplayName:  ptr("Quota A"),
 			Organization: servicequotav1.NewObjectReference("org-123", "", ""),
 			KafkaCluster: servicequotav1.NewObjectReference("lkc-1", "", ""),
 			Environment:  servicequotav1.NewObjectReference("env-1", "", ""),
-			Usage:        int32ToPtr(10),
-			AppliedLimit: int32ToPtr(15),
+			Usage:        ptr(int32(10)),
+			AppliedLimit: ptr(int32(15)),
 		}
 
 		quota2 := servicequotav1.ServiceQuotaV1AppliedQuota{
-			Id:           stringToPtr("quota_a"),
-			Scope:        stringToPtr("kafka_cluster"),
-			DisplayName:  stringToPtr("Qutoa A"),
+			Id:           ptr("quota_a"),
```

Comment:
Which one should we prefer? (I assume the following function exists)
```suggestion
			Id:           servicequotav1.PtrString("quota_a"),
```

Reply:
I definitely prefer to use the PtrString method from the sdk. I did a global find and replace for our in-house string to pointer methods which is probably why I didn't catch this.